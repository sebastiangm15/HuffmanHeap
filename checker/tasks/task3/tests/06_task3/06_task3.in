20
19 STLMZ
5 STLCH
39 STL84
14 STLAT
11 STL54
10 STLGV
14 STLXT
23 STLCN
35 STL07
48 STLXZ
5 STLU8
9 STLIX
30 STLSA
14 STLNH
25 STLGT
24 STLS2
29 STLWV
42 STLQV
47 STL2O
13 STLXS
10
STLQV
STL2O
STLGT
STL07
STLCN
STLS2
STLWV
STLSA
STLCH
STLNH
