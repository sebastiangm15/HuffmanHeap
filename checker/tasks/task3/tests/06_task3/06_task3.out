1111001011111000001011010101011110101010010
